### PR TITLE
Make copy packages documents more precise

### PR DESCRIPTION
**Pull request by TerryHowe**
Closes: https://github.com/aws/eks-anywhere/issues/4985
Closes: https://github.com/aws/eks-anywhere/issues/4983

### CODE REVIEW

File: docs/content/en/docs/reference/clusterspec/optional/registrymirror.md [hunk 1]
```diff
@@ -78,13 +78,21 @@ export REGISTRY_USERNAME=<username>
...
+Download and import EKS Anywhere images:
 ```bash
 eksctl anywhere download images -o eks-anywhere-images.tar
 docker login https://<private registry endpoint>
 ...
 eksctl anywhere import images -i eks-anywhere-images.tar
-eksctl anywhere copy packages --bundle ./eksa-bundle.yaml <private registry endpoint> --dst-cert rootCA.pem
 ```
+
+Download the EKS Anywhere bundle and use it to copy packages:
```
Comment (ivyostosh) [inline docs/content/en/docs/reference/clusterspec/optional/registrymirror.md]: If I start reading from line #73, it gets me confused of why I need to use the following commands to install packages. Wouldn't the download images and import images command suffice? Some clarification on the top description for this use case will be great.

Comment (TerryHowe) [reply docs/content/en/docs/reference/clusterspec/optional/registrymirror.md]: Okay, I added a sentence to summarize the process

**Comment by ivyostosh** [comment]
/hold
Until v0.14.2 release is done

**Pull request merged**